### ISSUE TITLE
Add mixed color support for LineEdit/TextEdit

### ISSUE THREAD

**Issue by qarmin** [slint-ui/slint#3728]
Currently I don't see any way to add colors to this widgets

This could be done by creating simpler bbcode that would allow to set only colors e.g.
```
text: "<red>Red text</red> <blue>blue text</blue>"
```

https://docs.godotengine.org/en/stable/tutorials/ui/bbcode_in_richtextlabel.html#using-bbcode

**Comment by tronical** [comment]
Yes, that's a missing feature at moment - we don't support rich text text :(

**Comment by tronical** [comment]
Duplicate of #2723
